### DISCUSSION TITLE
Is there a benchmark that compares the speed of axum vs actix?

### DISCUSSION THREAD

**Discussion by Cugatay** [tokio-rs/axum#1871] [Q&A]
Hey, I loved using Axum, it's well documented and very easy to use, while blllazingly fast. But I want to know the speed difference between actix and axum, is there a benchmark that compares them?

**Comment by davidpdrsn** [comment]
https://web-frameworks-benchmark.netlify.app/result

**Reply by Cugatay** [reply]
Thanks

**Reply by abhinavpatel0** [reply]
why actix is faster there than axum? and I also see some java thing coming on the top , how is this possible

**Reply by dayvejones** [reply]
It seems axum used to have higher results in this benchmark.
